hibp.js

```diff
@@ -9,6 +9,12 @@ const pkg = require("./package.json");
 
 
 const HIBP_USER_AGENT = `${pkg.name}/${pkg.version}`;
+// When HIBP "re-names" a breach, it keeps its old 'Name' value but gets a new 'Title'
+// We use 'Name' in Firefox (via Remote Settings), so we have to maintain our own mapping of re-named breaches.
```

Comment:
Firefox uses the data in the Remote Settings `Name` field. We manually edit the Remote Settings record to be `Name=Covve` instead of `Name=db8151dd`, even though HIBP still has `db8151dd` as the `Name`. :(

This PR is to fix the website, but I couldn't get a fix in for the `updatebreaches.js` script like I wanted.

Wanted to fix the 404 first.